x/twap/store.go

```diff
@@ -156,11 +156,6 @@ func (k Keeper) GetAllHistoricalTimeIndexedTWAPs(ctx sdk.Context) ([]types.TwapR
 	return osmoutils.GatherValuesFromStorePrefix(ctx.KVStore(k.storeKey), []byte(types.HistoricalTWAPTimeIndexPrefix), types.ParseTwapFromBz)
 }
 
-// getAllHistoricalPoolIndexedTWAPs returns all historical TWAPs indexed by pool id.
-func (k Keeper) getAllHistoricalPoolIndexedTWAPs(ctx sdk.Context) ([]types.TwapRecord, error) {
-	return osmoutils.GatherValuesFromStorePrefix(ctx.KVStore(k.storeKey), []byte(types.HistoricalTWAPPoolIndexPrefix), types.ParseTwapFromBz)
-}
```

Comment:
Err why are we deleting this, but re-implementing in export_test? Is a linter complaining about this being deadcode in non-test binaries?

Reply:
yeah, it is complaining about being unused because this is only used in tests.

I cannot move this into any `_test.go` file other than `export_test.go` because those files are `twap_test` package while `export_test.go` is still `twap`. In a separate package, methods cannot be implemented.

Reply:
gotcha :/, I wonder if we should just ignore the linter here

Reply:
Usually, I'm more supportive of removing unused code from production. However, this used to be production code until recently and is a short but useful helper that we are likely to reuse.

So I'm in agreement: [98f6617](https://github.com/osmosis-labs/osmosis/pull/6112/commits/98f6617b0133df3f4d048c94aa5d973ae9f3fcd7)